File: talks/2016-devcon-original/libgeth/accounts.go
package main

import "fmt"

// START OMIT
import "github.com/ethereum/go-ethereum/accounts"

func main() {
	am := accounts.NewManager("keystore", accounts.StandardScryptN, accounts.StandardScryptP)

	newAcc, _ := am.NewAccount("Creation password") // HL
	fmt.Println("New:", newAcc.Address.Hex())

	jsonAcc, _ := am.Export(newAcc, "Creation password", "Export password") // HL
	fmt.Println("Json:", string(jsonAcc))

	_ = am.DeleteAccount(newAcc, "Creation password") // HL
	fmt.Println("Accs:", am.Accounts())

	impAcc, _ := am.Import(jsonAcc, "Export password", "Import password") // HL
	fmt.Println("Imp:", impAcc.Address.Hex())
}

// END OMIT

File: talks/2016-devcon-original/libgeth/accounts.java
AccountManager am = Geth.newAccountManager(this.getFilesDir(), Geth.LightScryptN, Geth.LightScryptP);

Account newAcc = am.newAccount("Creation password"); // HL
textbox.append("New: " + newAcc.getAddress().getHex() + "\n\n");

byte[] jsonAcc = am.exportKey(newAcc, "Creation password", "Export password"); // HL
textbox.append("Json: " + new String(jsonAcc) + "\n\n");

am.deleteAccount(newAcc, "Creation password"); // HL
textbox.append("Accs: " + am.getAccounts().size() + "\n\n");

Account impAcc = am.importKey(jsonAcc, "Export password", "Import password"); // HL
textbox.append("Imp: " + impAcc.getAddress().getHex() + "\n\n");
